crates/dex-stable/src/lib.rs

```diff
@@ -1108,6 +1109,12 @@ pub mod pallet {
 }
 
 impl<T: Config> Pallet<T> {
+    /// only use two byte prefix to support 16 byte account id (used by test)
+    /// "modl" ++ "dex/stbl" is 12 bytes, and 4 bytes remaining for PoolId
+    pub fn pool_account_id(pool_id: &T::PoolId) -> T::AccountId {
+        T::PalletId::get().into_sub_account_truncating(pool_id)
```

Comment:
A bit of a nit, and not really directly related to this pr, but if `sizeof(poolId) > 4` or `sizeof(PalletId) > 12)` this doesn't really work I think (depending on endianness). But we're unlikely to change the runtime configuration so I guess it's fine..